EDMarketConnector.py

```diff
@@ -886,32 +891,37 @@ def export_market_data(self, data: CAPIData) -> bool:  # noqa: CCR001
 
         return True
 
-    def getandsend(self, event=None, retrying: bool = False):  # noqa: C901, CCR001
+    def capi_request_data(self, event=None) -> None:
         """
         Perform CAPI data retrieval and associated actions.
 
         This can be triggered by hitting the main UI 'Update' button,
         automatically on docking, or due to a retry.
+
+        :param event: Tk generated event details.
         """
+        logger.trace_if('capi.worker', 'Begin')
         auto_update = not event
         play_sound = (auto_update or int(event.type) == self.EVENT_VIRTUAL) and not config.get_int('hotkey_mute')
-        play_bad = False
-        err: Optional[str] = None
 
         if (
                 not monitor.cmdr or not monitor.mode or monitor.state['Captain']
                 or not monitor.system or monitor.mode == 'CQC'
         ):
+            logger.trace_if('capi.worker', 'CQC detected, aborting query')
+            # LANG: CAPI queries aborted because player is in CQC (Arena)
+            self.status['text'] = _('CQC detected, aborting CAPI query')
```

Comment:
But it also could be a mutlicrew, should it be clarified?

Reply:
Good point. I'll go fix that up.

Reply:
Ugh, I kind of think we want to test for things individually here, so as to have a specific message in each case.  More translation strings incoming!